Exercise/ex3-4-5-6-7.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Exercise 3, 4, 5, 6, 7</title>
+</head>
+<body>
+  <h2>Minh Dao T.</h2>
+  <button onclick="createEl()">Create Element</button>
+  <button onclick="removeEl()">Remove Element</button>
+  <button onclick="replaceEl()">Replace Element</button>
+  <button onclick="cloneN()">Clone Node</button>
+  <hr>
+  <ul id="myList"></ul>
+  <script>
+    let i = 0;
+    function createEl() {
+      i++;
+      let node = document.createElement("LI");
+      let txt  = document.createTextNode("Item" + i);
+      node.appendChild(txt);
+      document.getElementById("myList").appendChild(node);
```

Comment:
Naming convention
HTML attributes must be using kebab-case.
You should use a prefix to name for elements which are interactive with Javascript.
Example: 
`document.getElementById('js-my-list').appendChild(node);`
`js-` is prefix.